lib/shelley/src/Cardano/Wallet/Shelley/GenesisDataOrphans.hs

```diff
@@ -0,0 +1,197 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-missing-methods #-}
+{- HLINT ignore -}
+
+-- Module copied from cardano-node.
```

Comment:
Couldn't the module be imported instead so that the instances are put in Context?

Reply:
I got the sense that depending on cardano-node would be cumbersome due to all the dependencies.

Reply:
Done, wasn't that cumbersome, but needed to add a new Iohk-monitoring plugin to the snapshot.